src/java.base/share/classes/java/util/TimeZone.java

```diff
@@ -309,10 +309,10 @@ public void setID(String ID)
      * presentation to the user in the default locale.
      *
      * <p>This method is equivalent to:
-     * <blockquote><pre>
-     * getDisplayName(false, {@link #LONG},
-     *                Locale.getDefault({@link Locale.Category#DISPLAY}))
-     * </pre></blockquote>
+     * {@snippet lang=java :
+     * getDisplayName(false, LONG, // @link substring="LONG" target="#LONG"
```

Comment:
You could move those `@link` directives above the code and concatenate them with ":", e.g,
```
    * // @link substring="LONG" target="#LONG" :
    * getDisplayName(false, LONG,
```
so that the line won't be very long.

Reply:
Did not realize that could be done, made the fix, thank you